.travis.yml

```diff
@@ -7,19 +7,20 @@ env:
   - TOX_ENV=py27
   - TOX_ENV=py33
   - TOX_ENV=py34
+  - TOX_ENV=py35
 matrix:
-  include:
-    - python: 3.5
-      env:
-      - TOX_ENV=py35
+    include:
+        - python: 3.6
+          env:
+            - TOX_ENV=py36
```

Comment:
There's still some indentation that has snuck in here 😛 (basically just remove the spaces that show up in the diff)